backend/database/migrations/2022_11_16_134040_create_questions_table.php

```diff
@@ -15,7 +15,6 @@ public function up()
     {
         Schema::create('questions', function (Blueprint $table) {
             $table->uuid('id')->unique();
-            $table->string("question_id");
```

Comment:
the same thing apply here... use the update migration file to drop the column instead of deleting from the already created migration file.. revert this and drop the column in the update migration file

Reply:
alright